packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -417,7 +417,7 @@ class AppDomain extends Domain {
         );
         _sendAppEvent(app, 'stop');
       } catch (error) {
-        _sendAppEvent(app, 'stop', <String, dynamic>{'error': error.toString()});
+        _sendAppEvent(app, 'stop', <String, dynamic>{'error': _toJsonable(error)});
```

Comment:
There are other places in here where we were already returning `{'error': _toJsonable(error)}`.  How 'bout I update `_toJsonable()` to be error-aware?